src/panels/utilities/ClusterSpecCreationBuilder.ts

```diff
@@ -182,7 +142,3 @@ function generateNodeResourceGroup(resourceGroupName: string, clusterName: strin
 function removeWhitespace(str: string): string {
     return str.replace(/\s+/g, "");
 }
-
-function generateRbacName() {
```

Comment:
💡Thank you for this, will take a look into this soon. A key note here please, is this going to affect the way `Dev/test` cluster works? or this part of code only touches the `automatic` cluster? I did notice how `automatic` cluster don't work with Kaito at the moment.

Reply:
this method was only used for automatic cluster, it should not affect dev preset cluster.

Reply:
Thanks, sounds good!! I assume this is not the fix for the KAITO - install failing for `animatic cluster` asking here just to archive this as info. Thanks